src/IndodanaRequest.php

```diff
@@ -15,7 +15,7 @@ public static function createDefaultHeader($apiKey, $apiSecret)
     return [
       'Content-type: application/json',
       'Accept: application/json',
-      "Authorization: Bearer {$bearerToken}"
+      "Authorization: ${bearerToken}"
```

Comment:
The function name should be called `generateAuthorizationHeaderValue` if you want to use like this, `generateBearerToken` indicates that it will be paired with "bearer" complementary

Reply:
Alright!